vue/components/atoms/alert/alert.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <div class="alert" v-bind:style="alertStyle">
+        <div class="alert-content">
+            {{ message }}
+        </div>
+    </div>
+</template>
+
+<style scoped>
+.alert {
+    background-color: #151515;
+    box-shadow: 0 8px 50px 0 #61679426;
+    box-sizing: border-box;
+    display: block;
+    left: calc(50% - 127px);
+    opacity: 0.8;
+    padding: 23px 25px 23px 25px;
+    pointer-events: none;
+    position: absolute;
+    text-align: center;
+    top: calc(50% - 57px);
+    transition: opacity 0.65s ease-in-out;
+    user-select: none;
+    width: 305px;
+    z-index: 20;
+}
+
+.alert > .alert-content {
+    color: #ffffff;
+}
+</style>
+
+<script>
+export const Alert = {

```

Comment:
Even if the UI is different, ehis should be the same as https://github.com/ripe-tech/ripe-components-vue/blob/master/vue/components/ui/molecules/notification/notification.vue in terms of code. Any reason for not copy-pasting that one?